core/arch/arm/plat-imx/config/imx6q.h

```diff
@@ -0,0 +1,246 @@
+	defined(PLATFORM_FLAVOR_mx6dlsabresd) || \
+	defined(PLATFORM_FLAVOR_mx6qpsabresd)
+#define DRAM0_SIZE			0x40000000
+#endif
+
+#if defined(PLATFORM_FLAVOR_mx6qpsabreauto) || \
+	defined(PLATFORM_FLAVOR_mx6qsabreauto) || \
+	defined(PLATFORM_FLAVOR_mx6dlsabreauto)
+#define DRAM0_SIZE			0x80000000
+#endif
+
+/* Core number depends of SoC version. */
+#if defined(CFG_MX6QP) || defined(CFG_MX6Q)
+#define CFG_TEE_CORE_NB_CORE		4
+#endif
+#if defined(CFG_MX6D) || defined(CFG_MX6DL)
+#define CFG_TEE_CORE_NB_CORE		2
+#endif
+#if defined(CFG_MX6S)
+#define CFG_TEE_CORE_NB_CORE		1
+#endif
+
+/* Common RAM and cache controller configuration */
+#define CFG_TEE_RAM_VA_SIZE		(1024 * 1024)
+
+#define DDR_PHYS_START			DRAM0_BASE
+#define DDR_SIZE			DRAM0_SIZE
+
+#define CFG_DDR_START			DDR_PHYS_START
+#ifndef CFG_DDR_SIZE
+#define CFG_DDR_SIZE			DDR_SIZE
+#endif
+
+/*
+ * PL310 TAG RAM Control Register
+ *
+ * bit[10:8]:1 - 2 cycle of write accesses latency
+ * bit[6:4]:1 - 2 cycle of read accesses latency
+ * bit[2:0]:1 - 2 cycle of setup latency
+ */
+#ifndef PL310_TAG_RAM_CTRL_INIT
+#define PL310_TAG_RAM_CTRL_INIT		0x00000111
+#endif
+
+/*
+ * PL310 DATA RAM Control Register
+ *
+ * bit[10:8]:2 - 3 cycle of write accesses latency
+ * bit[6:4]:2 - 3 cycle of read accesses latency
+ * bit[2:0]:2 - 3 cycle of setup latency
+ */
+#ifndef PL310_DATA_RAM_CTRL_INIT
+#define PL310_DATA_RAM_CTRL_INIT	0x00000222
+#endif
+
+/*
+ * PL310 Auxiliary Control Register
+ *
+ * I/Dcache prefetch enabled (bit29:28=2b11)
+ * NS can access interrupts (bit27=1)
+ * NS can lockown cache lines (bit26=1)
+ * Pseudo-random replacement policy (bit25=0)
+ * Force write allocated (default)
+ * Shared attribute internally ignored (bit22=1, bit13=0)
+ * Parity disabled (bit21=0)
+ * Event monitor disabled (bit20=0)
+ * Platform flavor specific way config (dual / quad):
+ * - 64kb way size (bit19:17=3b011)
+ * - 16-way associativity (bit16=1)
+ * Platform flavor specific way config (dual lite / solo):
+ * - 32kb way size (bit19:17=3b010)
+ * - no 16-way associativity (bit16=0)
+ * Store buffer device limitation enabled (bit11=1)
+ * Cacheable accesses have high prio (bit10=0)
+ * Full Line Zero (FLZ) disabled (bit0=0)
+ */
+#ifndef PL310_AUX_CTRL_INIT
+#if defined(CFG_MX6QP) || defined(CFG_MX6Q) || defined(CFG_MX6D)
+#define PL310_AUX_CTRL_INIT		0x3C470800
+#else
+#define PL310_AUX_CTRL_INIT		0x3C440800
+#endif
+#endif
+
+/*
+ * PL310 Prefetch Control Register
+ *
+ * Double linefill disabled (bit30=0)
+ * I/D prefetch enabled (bit29:28=2b11)
+ * Prefetch drop enabled (bit24=1)
+ * Incr double linefill disable (bit23=0)
+ * Prefetch offset = 7 (bit4:0)
+ */
+#define PL310_PREFETCH_CTRL_INIT	0x31000007
+
+/*
+ * PL310 Power Register
+ *
+ * Dynamic clock gating enabled
+ * Standby mode enabled
+ */
+#define PL310_POWER_CTRL_INIT		0x00000003
+
+/*
+ * SCU Invalidate Register
+ *
+ * Invalidate all registers
+ */
+#define	SCU_INV_CTRL_INIT		0xFFFFFFFF
```

Comment:
Keep format consistent.

Reply:
Peng, i am not sure to understand.
are you talking about the space between define and SCU_INV_CTRL_INIT?

Reply:
the tabulation between `#define` and `SCU_INV_CTRL_INIT` should be replaced with a space.